printer_mods/ftobler/comb_cablecanal/README.md

```diff
@@ -0,0 +1,26 @@
+# Comb Cablecanal
+
+A cablecanal I designed to neaten up the spaghet.
```

Comment:
I guess this is intentional, and if so fine.

Reply:
Yes, it's intentional